opentreemap/importer/views.py

```diff
@@ -732,8 +746,23 @@ def export_single_import(request, instance, import_type, import_event_id):
     ie = _get_import_event(instance, import_type, import_event_id)
 
     if import_type == SpeciesImportEvent.import_type:
-        filename, csv_fields = "species.csv", fields.species.ALL
+        filename = "species.csv"
+        field_names = fields.species.ALL
+    else:
+        filename = "trees.csv"
+        field_names = ie.ordered_legal_fields()  # TODO: use ie's saved fields
+
+    return filename, get_import_export, (import_type, ie.pk,), field_names
+
+
+@task_output_as_csv
+def download_import_template(request, instance, import_type):
+    if import_type == SpeciesImportEvent.import_type:
+        filename = 'OpenTreeMap_Species_Import_Template.csv'
+        field_names = fields.species.capitalize(fields.species.ALL)
     else:
-        filename, csv_fields = "trees.csv", ie.ordered_legal_fields()
+        filename = 'OpenTreeMap_Tree_Import_Template.csv'
+        ie = TreeImportEvent(instance=instance)
+        field_names = ie.ordered_legal_fields_capitalized()
```

Comment:
Making this into a task is way overcomplicating things for a small fixed size CSV.  I think this is something that should be handled in the request/response cycle as part of a normal view.